tests/test_mapview.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from mapview import MapView
+
+
+class TextInputTest(unittest.TestCase):
+
+    def test_init_simple_map(self):
+        """
+        Makes sure we can initialize a simple MapView object.
+        """
+        kwargs = {}
+        mapview = MapView(**kwargs)
+        self.assertEqual(len(mapview.children), 2)
```

Comment:
Pretty basic test, but enough to raise `SyntaxError` issue mentioned in #51, #54 and #55. Subsequent pull request will fix the issue on Python3